script/Ciel/Day 7/07_01_CIEL07_5.txt

```diff
@@ -181,62 +185,62 @@ I don't think I did anything that justifies getting so worked up."
+// R: This one honestly doesn't bug me as much? You could simplify it to just "I'm well and truly at a loss for words"
+// R: if you want something that's emphasized and perhaps slightly more common I guess
+I can't believe she's maintaining her innocence even as new charges come to light.
+ So this is what it feels like to be at a loss for words.
 }
 [sha:b632bf47496109af3f4eb882c196fbf06b4ba555]{
 -- Page 446, Offset 15868.
 -- 「う……わ、わかった、わかりました、確かにカメラを壊したのは、余分な行為だったと思う」
-"Uh... O-Okay, okay. I suppose its true that the destruction of the cameras was excessive of me."
+"Um... O-Okay, you're right, maybe I went a %{i}little%{/i} overboard by breaking the cameras."
 }
 [sha:453ae2570433eeecd0d5165592d0a9afaa219b9d]{
 -- Page 447, Offset 15869. Mods: @n.
 -- 「要はアレね。人間的に、きちんと玄関から行けば良かったんでしょう？　次からはそうするわ。
-"So basically, it's like, it'd been fine――for humans, at least――if I just came through the front gate, right?
- I'll do that next time then.
+"Anyway, I get where I went wrong.
+ I should have just come in through the front gate like a human would, right?
+ I'll be sure to do it that way next time.
 }
 [sha:2328b3c428c04aa3f4b3e65126afb9e1e4f7fbbf]{
 -- Page 447, Offset 15870. Glued.
 -- 招かれないと家に入れないなんて特性もないし―――
 // talking about that old vampire adage about how they can only enter houses they've been invited, so I'm using weakness to make that more clear.
-After all, I don't have that weakness of being unable to enter someone's house without first being invited――
+ After all, I don't have that weakness of being unable to enter someone's house without first being invited―――
 }
 [sha:f77b8ecf2781d21428a0daf8fe794281b5d507fb]{
 -- Page 447, Offset 15871.
 -- ちょっ、待って、どこ行くの志貴!?　話、まだ終わってないよ!?」
-Hey, wait, where are you going, Shiki?! I'm not done talking yet?!"
+Hey, wait, where are you going, Shiki?! I'm not done talking yet!"
 }
 [sha:59b0ede2509e23126f0d523291275b29e980fbcb]{
 -- Page 448, Offset 15872.
 -- おお。足が無意識に帰路についていた。
-Whoa, my legs unconsciously started making themselves their way home.
+Whoa, my legs just started making their own way home.
 }
 [sha:3a12377203af20a3e361d046ebd048014b547a0c]{
 -- Page 448, Offset 15873.
 -- アルクェイドの発言に、いよいよ本能が危険信号を発したと見える。
-That could be interpreted as my instincts finally signaling danger upon hearing Arcueid talking.
+Seems like what Arcueid just said finally set some alarm bells ringing in my subconscious.
 }
 [sha:2504f729b8daf42d6fcab244c25036ccf2e2a332]{
 -- Page 448, Offset 15874.
 -- ……とはいっても逃げ切れる訳がないので、テクテクとアルクェイドのもとに戻る。
-That said,
- it's not like I could get away from her,
- so I plotter back over to Arcueid's side.
+Though... it's not like I'd actually be able to give her the slip,
+ so I trudge back over to where Arcueid is standing.
 }
 [sha:b4f838845f5865d09dded09f17aa2c73798ce960]{
 -- Page 449, Offset 15875.
 -- 「わるい、考え事をしていたら足が勝手に動いた。
-"My bad, as I was thinking my legs took a life on their own.
+"My bad, as I was thinking my legs took on a life on their own.
```

Comment:
Wouldn't it be "took on a life **of** their own?"

Though I suppose it's my bad for the messy writing to begin with.